docs/src/ref_impl/reference_implementations.md

```diff
@@ -3,24 +3,26 @@
     SPDX-License-Identifier: CC-BY-SA-4.0
 -->
 
-# Build Configurations
+# Reference Implementations
 
-Our hardened operating system (OS) targets are build configurations based on NixOS. The canonical URL for the upstream git repository is: [https://github.com/NixOS](https://github.com/NixOS).
+Our hardened OS targets are build configurations based on NixOS. The canonical URL for the upstream Nix git repository is: [https://github.com/NixOS](https://github.com/NixOS).
+
+Build configurations define our dependencies and configuration changes to packages and build mechanisms of NixOS. If you want to try Ghaf, see [Build and Run](../ref_impl/build_and_run.md).
 
-Build configurations define our dependencies and configuration changes to packages and build mechanisms of NixOS. If you want to try, see the [reference implementations](../build_config/reference_implementations.md).
 
 ## Approach
 
-A build configuration is a target to build our hardened OS for a particular hardware device. Most packages used in a build configuration come from [nixpkgs - NixOS Packages collection](https://github.com/NixOS/nixpkgs).
+A build configuration is a target to build the hardened OS for a particular hardware device. Most packages used in a build configuration come from [nixpkgs—NixOS Packages collection](https://github.com/NixOS/nixpkgs).
```

Comment:
My earlier comment related to m-dash was preferring the more open format with spaces instead of removing the spaces like here.